lib/AST/ASTPrinter.cpp

```diff
@@ -5134,12 +5158,35 @@ void PrintAST::visitBinaryExpr(BinaryExpr *expr) {
 }
 
 void PrintAST::visitCoerceExpr(CoerceExpr *expr) {
+  visit(expr->getSubExpr());
+  Printer << " as ";
+  printType(expr->getCastType());
 }
 
-void PrintAST::visitOneWayExpr(OneWayExpr *expr) {
-}
+void PrintAST::visitOneWayExpr(OneWayExpr *expr) { }
 
 void PrintAST::visitClosureExpr(ClosureExpr *expr) {
+  Printer << "{ ";
```

Comment:
Today I learnt we also support attributes for closures, which we also ought to print here by printing its `getAttrs()`, this allows e.g `{ @MainActor in }`